test/letters/reducers/index.unit.spec.js

```diff
@@ -38,71 +65,74 @@ describe('letters reducer', () => {
   it('should handle failure to fetch letters', () => {
```

Comment:
Might want to include a test to make sure the reducer returns `initialState` when it isn't passed an action? This would test the default case in the switch.

Reply:
:man_shrugging: 

Why not?